src/test/java/cat/udl/eps/softarch/academicrecruit/steps/CommitteeStepDefs.java

```diff
@@ -0,0 +1,51 @@
+package cat.udl.eps.softarch.academicrecruit.steps;
+
+import cat.udl.eps.softarch.academicrecruit.domain.Committee;
+import cat.udl.eps.softarch.academicrecruit.repository.AdminRepository;
+import cat.udl.eps.softarch.academicrecruit.repository.ApplicantRepository;
+import cat.udl.eps.softarch.academicrecruit.repository.CommitteeRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.When;
+import org.json.JSONObject;
+import org.springframework.http.MediaType;
+
+import static org.hamcrest.Matchers.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+
+public class CommitteeStepDefs {
+    final StepDefs stepDefs;
+    final CommitteeRepository committeeRepository;
+
+    CommitteeStepDefs(StepDefs stepDefs, CommitteeRepository committeeRepository, AdminRepository adminRepository) {
+        this.stepDefs = stepDefs;
+        this.committeeRepository = committeeRepository;
+    }
+
+    @When("I assign a rank {string} to a user")
```

Comment:
This approach, not specifying a user, makes sense for the moment as you don't have relationships among entities yet. However, once they are part of the model, it should be defined.

By the way, from the model, a Committee just has 1 user with the corresponding rank. Thus, maybe it is better to call it Committee participation?